snap/hooks/configure

```diff
@@ -10,6 +10,5 @@ fi
 
 GPIOCHIP=$(snapctl get gpiochip)
 if (( GPIOCHIP != 0 && GPIOCHIP != 4 )); then
-    logger -t $TAG --stderr "gpiochip: '$GPIOCHIP' is not supported; set to 0 or 4"
-    exit 1
+    logger -t $TAG --stderr "WARNING! gpiochip: Unless that is being used for mocking, '$GPIOCHIP' is not supported; set to 0 or 4"
```

Comment:
I believe the standard error will not be printed to the user's console unless the hook exits with a non-zero code.

A better approach could be to dedicate another configuration option to skip number checking. Ideally something not tied to tests but rather generic and useful beyond the testing scope, such as when running the apps on a device with another chip number where we want to test this snap without confinement. E.g. key could be `gpiochip-validation=true`, defaulting to true (self-documented) or `skip-gpiochip-validation` defaulting to unset/empty string.

Reply:
Fixed! Thank you!